infinitered/ReactNativeEssentials
# Chapter 8: Blending In With Surroundings (Light and Dark Mode)

### Summary

Chapter 8: Blending In With Surroundings (Light and Dark Mode)

This PR implements our themeing.

### How to verify

1. `yarn`
2. `yarn start`
3. `i` & `a`
4. Toggle dark/light mode using: cmd+shift+a

**Expected results**: app toggles between light and dark mode

### Screenshots

| Light                                             | Dark                                             |
| ----------------------------------------------- | --------------------------------------------------- |
| <img width="300" alt="ios light screenshot" src="https://github.com/user-attachments/assets/1dab70d8-249b-4287-a7b0-093d014a989a" /> | <img width="300" alt="ios dark screenshot" src="https://github.com/user-attachments/assets/648c35f4-a14e-4e5f-897f-40ad60b3be1c" /> |
